src/veins/modules/mobility/traci/TraCIMobility.cc

```diff
@@ -64,6 +67,7 @@ void TraCIMobility::Statistics::recordScalars(cSimpleModule& module)
     if (maxSpeed != -MY_INFINITY) module.recordScalar("maxSpeed", maxSpeed);
     module.recordScalar("totalDistance", totalDistance);
     module.recordScalar("totalCO2Emission", totalCO2Emission);
+    module.recordScalar("totalFuelConsumption", totalFuelConsumption);
```

Comment:
Can this scalar be derived from the newly introduced signals? This would avoid the for computing recorded values manually.